scripts/create_service_principal.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+#echo "Usage:
+#  1 bash create-service-principal.sh
+#  2 bash create-service-principal.sh <Subscription ID>
+# You need Azure CLI: https://docs.microsoft.com/azure/xplat-cli-install
```

Comment:
I recommend to use az cli, instead of xplat-cli-install. 
Check, https://github.com/Azure/azure-cli